### PR TITLE
USWDS-Site - Changelog: Add entry for Input mask fix hover state [USWDS#5378]

### PR DESCRIPTION
**Pull request by amyleadem**
# Summary
Added changelog entries for uswds/uswds#5378

## Related PR

Related to uswds/uswds#5378
> Fixed a bug that caused the input mask hover state to adopt disabled styling.

## Preview link
[Input mask changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-5378/components/input-mask/#latest-updates)

### CODE REVIEW

File: _data/changelogs/component-input-mask.yml [hunk 1]
```diff
@@ -2,6 +2,20 @@ title: Input mask
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Fixed a bug that caused the hover state to adopt disabled styling.
+    summaryAdditional:
```
Comment (mahoneycm) [inline _data/changelogs/component-input-mask.yml:7]: Should we remove empty items? I believe other PRs exclude `summaryAdditional` when it is not needed.

```suggestion

```

File: _data/changelogs/component-input-mask.yml [hunk 2]
```diff
@@ -2,6 +2,20 @@ title: Input mask
...
+    affectsStyles: true
+    githubPr: 5378
+    githubRepo: uswds
+    versionUswds: N.N.N
+  - date: NNNN-NN-NN
+    summary: Improved legibility in forced colors mode.
+    summaryAdditional:
```
Comment (mahoneycm) [inline _data/changelogs/component-input-mask.yml:14]: IF we remove, we should remove this one as well.

```suggestion

```

**Review by mahoneycm — COMMENTED** [review]
Looks good! Just a comment about potentially removing the empty `summaryAdditional` items 👍

**Comment by amyleadem** [comment]
@mahoneycm I removed the empty `summaryAdditional` fields. I think it is a good idea to do so moving forward. Thanks!

**Review by mahoneycm — APPROVED** [review]
Awesome! lgtm!

**Review by mejiaj — APPROVED** [review]

**Review by thisisdano — APPROVED** [review]

**Pull request merged**